Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM node:10
+WORKDIR /src
+
+RUN echo $(grep "VERSION=" /etc/os-release | cut -d "(" -f2 | cut -d ")" -f1) | \
+  xargs -i echo "deb http://apt.postgresql.org/pub/repos/apt/ {}-pgdg main" > /etc/apt/sources.list.d/postgresql.list && \
+  wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
+
+# install PostgreSQL 9.5
+RUN apt-get update && apt-get -y upgrade && \
+  apt-get install --no-install-recommends -y \
+  postgresql-client-9.5 && \
+  apt-get clean
+
+# install Azure Functions Core Tools 3
+RUN apt-get install -y apt-transport-https && \
+  curl https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > microsoft.gpg && \
+  mv microsoft.gpg /etc/apt/trusted.gpg.d/microsoft.gpg && \
+  apt-get install -y lsb-release && \
```

Comment:
it might be worth using the `VERSION` like we do to install og vs install the `lsb_release` package here...

Reply:
Proved to be too much time to get that to work, I'd need to pass in two separate arguments to the `sh -c 'echo "deb [arch=amd64] ...` command and for some reason using variables did not work (not sure why) - 
```sh
  OS_VERSION_ID=$(grep "VERSION_ID=" /etc/os-release | cut -d '"' -f2) \
  OS_CODENAME=$(grep "VERSION=" /etc/os-release | cut -d "(" -f2 | cut -d ")" -f1); && \
  sh -c 'echo "deb [arch=amd64] https://packages.microsoft.com/debian/${OS_VERSION_ID}/prod ${OS_CODENAME} main" > /etc/apt/sources.list.d/dotnetdev.list'
```
Variable values showed up as blank when I tried this out.

Will be just sticking with `lsb_release`.